patches/net/minecraft/server/Bootstrap.java.patch

```diff
@@ -9,7 +9,12 @@
                      wrapStreams();
                      bootstrapDuration.set(Duration.between(instant, Instant.now()).toMillis());
                  }
-@@ -121,7 +_,6 @@
+@@ -117,11 +_,11 @@
+     public static void validate() {
+         checkBootstrapCalled(() -> "validate");
+         if (SharedConstants.IS_RUNNING_IN_IDE) {
++            if (false) // Neo: this can be very annoying
```

Comment:
Why not check for translations?

Reply:
It could be annoying, but if it's wanted i can remove that patch, I don't really care.

Reply:
might as well remove it until people complain about it, imo

Reply:
Maybe put it behind a separate flag?

I could see this being very useful, but as something you'd turn on and run once or twice when you want it then turn it off the rest of the time.